app/rpc/RPC.ts

```diff
@@ -97,24 +97,29 @@ export default class RPC {
   }
 
   static async rpc_setInterruptSyncAfterBatch(value: string): Promise<void> {
-    const resp = await RPCModule.execute('interrupt_sync_after_batch', value);
-    console.log('interrupt sync', value, resp);
+    await RPCModule.execute('interrupt_sync_after_batch', value);
+    //console.log('interrupt sync', value, resp);
   }
 
   static async rpc_getZecPrice(): Promise<number> {
+    // values:
+    // 0   - initial/default value
+    // -1  - error in Gemini/zingolib/RPCModule.
+    // > 0 - real value
     const resultStr: string = await RPCModule.execute('updatecurrentprice', '');
     //console.log(resultStr);
 
     if (resultStr) {
-      if (resultStr.toLowerCase().startsWith('error')) {
+      if (resultStr.toLowerCase().startsWith('error') || isNaN(parseFloat(resultStr))) {
         //console.log(`Error fetching price ${resultStr}`);
-        return 0;
+        return -1;
       } else {
         return parseFloat(resultStr);
       }
+    } else {
+      //console.log(`Error fetching price ${resultStr}`);
+      return -1;
```

Comment:
differs from this value, but it probably doesnt matter

Reply:
Thanks for the explanation, now, I got it... I'm going to change it with two different texts in the info screen.